### PR TITLE
RD-6318 CLI documentation for Secrets Providers

### PR DESCRIPTION
**Pull request by adrian-polanczyk-codilime**

### CODE REVIEW

File: content/cli/orch_cli/secrets-providers.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+private_resource:          False
+
+
+...
+{{< /highlight >}}
+
+### list
+
+#### Usage
+`fy secrets providers list [OPTIONS]`
```
Comment (tehasdf) [inline content/cli/orch_cli/secrets-providers.md:119]:
```suggestion
`cfy secrets providers list [OPTIONS]`
```

**Review by tehasdf — DISMISSED** [review]

File: content/working_with/manager/using-secrets.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ All the values of the secrets are encrypted in the database. When you create a s
 A secret with a hidden value means the value is only shown to the user who created it, tenant managers and sys-admins.
 Users can use the secret according to the user roles and the visibility of the secret.
 
+### Secrets from remote provider
+
+Secret value can be provided by remote "Secrets Provider", like Vault or another Cloudify Manager.
+If a provider is set up for a secret resource, and provider connection is available, then a secret value will be passed in decrypted form.
+More about Secrets Providers [here](https://github.com/cloudify-cosmo/docs.getcloudify.org/blob/master/content/cli/orch_cli/secrets-providers.md).
```
Comment (tehasdf) [inline content/working_with/manager/using-secrets.md:25]: can't be just a link, needs to be `relref`

File: content/working_with/manager/using-secrets.md [hunk 3]
```diff
@@ -18,6 +18,11 @@ All the values of the secrets are encrypted in the database. When you create a s
 A secret with a hidden value means the value is only shown to the user who created it, tenant managers and sys-admins.
 Users can use the secret according to the user roles and the visibility of the secret.
 
+### Secrets from remote provider
+
+Secret value can be provided by remote "Secrets Provider", like Vault or another Cloudify Manager.
```
Comment (tehasdf) [inline content/working_with/manager/using-secrets.md:23]: don't say `Cloudify Manager` but rather use the macros, `{{< param product_name >}}` = `Cloudify`,  `{{< param cfy_manager_name >}}` = `Cloudify Manager`

This is for whitelabeling, where some customers relabel the manager with their own name

File: content/cli/orch_cli/secrets-providers.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
+---
+title: secrets providers
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+aliases: /cli/secrets-providers/
+---
+
+The `cfy secrets providers` command is used to manage Cloudify Secrets Providers.
```
Comment (tehasdf) [inline content/cli/orch_cli/secrets-providers.md:9]: ditto `Cloudify`

File: content/working_with/manager/using-secrets.md (hunk 3 above)
Comment (tehasdf) [reply content/working_with/manager/using-secrets.md:23]: (btw, the available params are listed in config.toml)

**Comment by tehasdf** [comment]
staging build is here https://docs.cloudify.co/staging/RD-6318-secrets-providers-cli-docs/working_with/manager/using-secrets/#secrets-from-remote-provider

**Review by tehasdf — APPROVED** [review]

**Review by alexmolev — APPROVED** [review]

**Pull request merged**